### PR TITLE
fix(CI): add `DEBIAN_FRONTEND=noninteractive` in arm64 jobs

### PR DESCRIPTION
**Pull request by Andreagit97**
**What type of PR is this?**

/kind bug

**Any specific area of the project related to this PR?**

/area CI

**Does this PR require a change in the driver versions?**

No

**What this PR does / why we need it**:

For some reason yesterday we didn't need `DEBIAN_FRONTEND=noninteractive` in our jobs but today `circleCI` wants it otherwise the job will face a context deadline exceeded

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -10,8 +10,8 @@ jobs:
       - run:
           name: Install deps ⛓️
           command: |
-            sudo apt update
-            sudo apt install -y --no-install-recommends ca-certificates cmake build-essential git pkg-config autoconf automake libelf-dev libcap-dev linux-headers-$(uname -r) clang-14 llvm-14
+            sudo apt update -y
+            sudo DEBIAN_FRONTEND=noninteractive apt install -y --no-install-recommends ca-certificates cmake build-essential git pkg-config autoconf automake libelf-dev libcap-dev linux-headers-$(uname -r) clang-14 llvm-14
```
Comment (FedeDP) [inline .circleci/config.yml:14]: Classic :rofl:

**Review by FedeDP — APPROVED** [review]
/approve

**Review by jasondellaluce — APPROVED** [review]
/approve

**Pull request merged**